src/ray/core_worker/core_worker.cc

```diff
@@ -281,13 +283,35 @@ void CoreWorker::ReportActiveObjectIDs() {
   heartbeat_timer_.async_wait(boost::bind(&CoreWorker::ReportActiveObjectIDs, this));
 }
 
-void CoreWorker::PromoteObjectToPlasma(const ObjectID &object_id) {
+void CoreWorker::PromoteToPlasmaAndGetOwnershipInfo(const ObjectID &object_id,
+                                                    TaskID *owner_id,
+                                                    rpc::Address *owner_address) {
   RAY_CHECK(object_id.IsDirectCallType());
   auto value = memory_store_->GetOrPromoteToPlasma(object_id);
   if (value != nullptr) {
     RAY_CHECK_OK(
         plasma_store_provider_->Put(*value, object_id.WithPlasmaTransportType()));
   }
+
+  auto has_owner = reference_counter_->GetOwner(object_id, owner_id, owner_address);
```

Comment:
Is this a bool? If not, `has_owner` seems misleading.

Reply:
It actually is a bool.